modules/nw-controlling-dns-pod-placement.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/dns-operator.adoc
+
+[id="nw-controlling-dns-pod-placement_{context}"]
+= Controlling DNS pod placement
+
+The DNS Operator has two daemon sets, one for CoreDNS and one for managing `/etc/hosts`. The daemon set for `etc/hosts` must run on every node host and include an entry for the cluster image registry to support pulling images. Security policies can prohibit communication between pairs of nodes, which prevents the daemon set for CoreDNS from running on every node.
+
+As a cluster administrator, you can use a custom node selector to configure the daemon set for CoreDNS to run or not run on certain nodes.
+
+
+.Prerequisites
+
+* You installed the `oc` CLI.
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+* Configure the `spec.nodePlacement.nodeSelector` API field to prevent communication between certain nodes.
```

Comment:
Since these unordered steps appear to be two different options (though I might be misunderstanding), maybe flip around the sentences to lead with the different actions? So it's more immediately clear from a glance which is which. For example, for this bullet here something like:

```
* To prevent communication between certain nodes, configure the `spec.nodePlacement.nodeSelector` API field:
```

and likewise for the later unordered item:

```
* To allow the daemon set for CoreDNS to run on nodes, configure a taint and toleration:
```

Reply:
They are two different options, thank you.